stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/EmojiUtil.kt

```diff
@@ -44,6 +43,19 @@ public object EmojiUtil {
         return isEmojiOnly(message) && message.text.replaceFirst(EMOJI_REGEX, "").isEmpty()
     }
 
+    /**
+     * Counts the number of emoji inside a message.
+     *
+     * @return The number of emojis inside a message.
+     */
+    public fun getEmojiCount(message: Message): Int {
```

Comment:
I think something like this would be more readable and understandable. Probably more performant too (wild guess).

```kotlin
    public fun getEmojiCount(message: Message): Int =
            EMOJI_REGEX.findAll(message.text).count()
```

Reply:
Seems right, will replace it with this